<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wifiStartup</class>
 <widget class="QDialog" name="wifiStartup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="headline">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Start WiFi-Setup</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QPlainTextEdit" name="scriptOutput">
     <property name="toolTip">
      <string>Messages produced by the startup script</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="startScriptButton">
     <property name="toolTip">
      <string>Start the setup script to set up the wifi interface</string>
     </property>
     <property name="text">
      <string>Start Scipt</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/startscript.svg</normaloff>:/icons/icons/startscript.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+S, Ctrl+W</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="exportScriptButton">
     <property name="toolTip">
      <string>Export the startup script for external use!</string>
     </property>
     <property name="text">
      <string>export Script</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/export.svg</normaloff>:/icons/icons/export.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>247</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="doneButton">
     <property name="toolTip">
      <string>Close this window.</string>
     </property>
     <property name="text">
      <string>done</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/done.svg</normaloff>:/icons/icons/done.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startScriptButton</sender>
   <signal>clicked()</signal>
   <receiver>wifiStartup</receiver>
   <slot>slot_startScriptButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportScriptButton</sender>
   <signal>clicked()</signal>
   <receiver>wifiStartup</receiver>
   <slot>slot_exportScriptButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doneButton</sender>
   <signal>clicked()</signal>
   <receiver>wifiStartup</receiver>
   <slot>slot_doneButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_startScriptButtonClicked()</slot>
  <slot>slot_exportScriptButtonClicked()</slot>
  <slot>slot_doneButtonClicked()</slot>
 </slots>
</ui>
